rskj-core/src/test/java/co/rsk/peg/whitelist/WhitelistSupportImplTest.java

```diff
@@ -66,43 +68,44 @@ void getLockWhitelistSize_whenLockWhitelistHasEntries_shouldReturnOne() {
 
     @Test
     void getLockWhitelistEntryByIndex_whenLockWhitelistIsEmpty_shouldReturnNull() {
```

Comment:
You'll need to rename these tests. The name says `shouldReturnNull` not it should be `shouldReturnEmpty`